script/deploy

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-
+script_dir=$(cd `dirname $0`; pwd)
+echo cru$script_dir
+base_dir=`dirname ${script_dir}`
+echo build $base_dir
+echo $git_book_dir
 git clone https://github.com/apache/incubator-servicecomb-docs.git
```

Comment:
It looks the script won't check the change the servicecomb doc, and just rebuild the whole document once the site doc is changed.

Reply:
cancel the gitbook building logic